### PR TITLE
fix cancelable focus, make confirm discard button red

### PR DESCRIPTION
**Pull request by nelix**
screen shot 2015-02-23 at 7 51 27 pm

Also adds a title to the cancel dialog

### CODE REVIEW

File: packages/cancelable-edit/example.jsx [hunk 1]
```diff
@@ -27,16 +27,31 @@ module.exports = React.createClass({
...
+
+      <pre>
+{`
+  <CancelableEdit
+    creating
+    saveButtonText="Create"
+    placeholder="Yo..."
+  />
+`}
+      </pre>
```
Comment (grapegravity) [inline packages/cancelable-edit/example.jsx]: Let’s move this kind of thing to the README from now on.

**Comment by nelix** [comment]
I'll get a README together once I finish messing with it in Stack, is that cool?

File: packages/cancelable-edit/index.jsx [hunk 2]
```diff
@@ -211,7 +206,8 @@ module.exports = React.createClass({
                 success
                 onClick={this.handleSave}
               >
-              {this.props.saveButtonText || 'Save ' + this.props.displayName}
+              {this.props.saveButtonText ||
+                'Save ' + (this.props.displayName || '')}
```
Comment (grapegravity) [inline packages/cancelable-edit/index.jsx]: Worth using an ES6 template string here?

Comment (nelix) [inline packages/cancelable-edit/index.jsx]: I wonder if any code is actually using displayName, if not I'd just delete displayName handling...

Comment (nelix) [inline packages/cancelable-edit/index.jsx]: I mean, there can't be any really, I didnt get a change to even use this before I changed the API

**Pull request merged**